mava/systems/tf/maddpg/system.py

```diff
@@ -94,89 +105,71 @@ def __init__(
+                feedforward or recurrent.
+            num_executors: number of executor processes to run in
+                parallel..
+            environment_spec: description of
                 the action, observation spaces etc. for each agent in the system.
-                Defaults to None.
-            shared_weights (bool, optional): whether agents should share weights or not.
-                When agent_net_keys are provided the value of shared_weights is ignored.
-                Defaults to True.
-            agent_net_keys: (dict, optional): specifies what network each agent uses.
-                Defaults to {}.
-            discount (float, optional): discount factor to use for TD updates. Defaults
-                to 0.99.
-            batch_size (int, optional): sample batch size for updates. Defaults to 256.
-            prefetch_size (int, optional): size to prefetch from replay. Defaults to 4.
-            target_averaging (bool, optional): whether to use polyak averaging for
-                target network updates. Defaults to False.
-            target_update_period (int, optional): number of steps before target
-                networks are updated. Defaults to 100.
-            target_update_rate (Optional[float], optional): update rate when using
-                averaging. Defaults toNone.
-            executor_variable_update_period (int, optional): number of steps before
-                updating executor variables from the variable source. Defaults to 1000.
-            min_replay_size (int, optional): minimum replay size before updating.
-                Defaults to 1000.
-            max_replay_size (int, optional): maximum replay size. Defaults to 1000000.
-            samples_per_insert (Optional[float], optional): number of samples to take
-                from replay for every insert that is made. Defaults to 32.0.
-            policy_optimizer (Union[ snt.Optimizer, Dict[str, snt.Optimizer] ],
-                optional): optimizer(s) for updating policy networks. Defaults to
-                snt.optimizers.Adam(learning_rate=1e-4).
-            critic_optimizer (snt.Optimizer, optional): optimizer for updating critic
-                networks. Defaults to snt.optimizers.Adam(learning_rate=1e-4).
-            n_step (int, optional): number of steps to include prior to boostrapping.
-                Defaults to 5.
-            sequence_length (int, optional): recurrent sequence rollout length. Defaults
-                to 20.
-            period (int, optional): Consecutive starting points for overlapping
-                rollouts across a sequence. Defaults to 20.
-            bootstrap_n (int, optional): Used to determine the spacing between
+            trainer_networks: networks each
+                trainer trains on.
+            network_sampling_setup: List of networks that are randomly
+                sampled from by the executors at the start of an environment run.
```

Comment:
For clarity, can we please add another sentence on this. Something like defaults to `fixed_agent_networks`, which means ....  . Essentially so that people who don't need to work about this var, can ignore it, while others can tweak it. I know you do mention this below, but I think it would be useful here.

Reply:
Agreed. I updated the docstring.